packages/block-library/src/quote/transforms.js

```diff
@@ -74,44 +153,47 @@ const transforms = {
 				return (
 					node.nodeName === 'BLOCKQUOTE' &&
 					// The quote block can only handle multiline paragraph
-					// content with an optional cite child.
-					Array.from( node.childNodes ).every(
-						isParagraphOrSingleCite
-					)
+					// content with an optional figcaption child.
+					Array.from( node.childNodes ).every( isAllowedNode )
 				);
 			},
-			schema: ( { phrasingContentSchema } ) => ( {
-				blockquote: {
-					children: {
-						p: {
-							children: phrasingContentSchema,
-						},
-						cite: {
-							children: phrasingContentSchema,
-						},
-					},
-				},
-			} ),
+			transform: ( node ) =>
+				createBlock(
+					'core/quote',
+					{},
+					rawHandler( {
+						HTML: node.innerHTML,
+						mode: 'BLOCKS',
+					} )
+				),
 		},
 	],
 	to: [
+		{
+			type: 'block',
+			blocks: [ 'core/group' ],
+			transform: ( { attribution }, innerBlocks ) =>
+				createBlock(
+					'core/group',
+					{},
+					attribution
+						? [
+								...innerBlocks,
+								createBlock( 'core/paragraph', {
+									content: attribution,
+								} ),
+						  ]
+						: innerBlocks
+				),
+		},
 		{
 			type: 'block',
 			blocks: [ 'core/paragraph' ],
-			transform: ( { value, citation } ) => {
-				const paragraphs = [];
-				if ( value && value !== '<p></p>' ) {
-					paragraphs.push(
-						...split(
-							create( { html: value, multilineTag: 'p' } ),
-							'\u2028'
-						).map( ( piece ) =>
-							createBlock( 'core/paragraph', {
-								content: toHTMLString( { value: piece } ),
-							} )
-						)
-					);
-				}
+			transform: ( { citation }, innerBlocks ) => {
+				const paragraphs = toBlocksOfType(
```

Comment:
I think this transform no longer makes sense. We should allow the user to unquote, but transforming all nested context to paragraphs is very strange.

Reply:
Yeah, I think this transform could be removed and an "Unquote" button could be added to the Quote's toolbar.